extensions/remove-paywall/package.json

```diff
@@ -20,6 +20,12 @@
       "description": "Remove the paywall from the selected or clipboard URL.",
       "mode": "no-view",
       "arguments": [
+        {
+          "name": "url",
+          "type": "text",
+          "placeholder": "URL",
+          "required": false
```

Comment:
Making it optional makes sense since the value can be taken from the selected text or clipboard, but the user doesn't know it.

Have you tried restoring the additional hint in the placeholder, like "URL (or selected text/clipboard)". Does it fit the input field? If it looks nice, I suggest doing so until Raycast API doesn't support prefilling input fields.

Reply:
Sadly the text is cut off. I tried "URL (or selection/clipboard)", but the placeholder is still to long.

![Raycast 2024-01-23 at 12 39 46](https://github.com/raycast/extensions/assets/69514484/73bf6dd4-5687-4906-8253-04e3b8d156d3)

Should I leave it like this or do you have a different idea for a shorter text that fits?

Reply:
I guess this is our best option at the moment 🤷 so please leave it like this.

Thank you @DannySeidel